service/src/main/java/datawave/microservice/authorization/AuthorizationOperationsV2.java

```diff
@@ -0,0 +1,71 @@
+package datawave.microservice.authorization;
+
+import datawave.microservice.authorization.user.ProxiedUserDetails;
+import datawave.security.authorization.CachedDatawaveUserService;
+import datawave.security.authorization.DatawaveUser;
+import datawave.security.authorization.JWTTokenHandler;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.bus.BusProperties;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.security.RolesAllowed;
+
+/**
+ * Presents the REST operations for the authorization service. This version returns the updated (V2) DatawaveUser
+ */
+@RestController
+@RequestMapping(path = "/v2", produces = MediaType.APPLICATION_JSON_VALUE)
+public class AuthorizationOperationsV2 extends AuthorizationOperationsV1 {
```

Comment:
3 of 8 methods are overridded.  I considered duplicating everything, but generally avoid that because future bug fixes would have to be made in multiple places.  Also considered composition, but inherritance seemed cleaner here.
If v1 gets deprecated/removed, we can move the non-overridden methods to v2 in the process.